### PR TITLE
adds a line to the syndie fedora so people know what the fuck it does

### PR DESCRIPTION
**Pull request by Skrem7**
# Document the changes in your pull request

People don't know what this does SO MUCH SO that a bug report was submitted! FUCKING CRAZY!

# Changelog

:cl:  
spellcheck: Syndie fedora has a description change to say you can extend its blades now
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -103,7 +103,7 @@
 /obj/item/clothing/head/det_hat/evil
 	name = "suspicious fedora"
 	icon_state = "syndicate_fedora"
-	desc = "A suspicious black fedora with a red band."
+	desc = "A suspicious black fedora with a red band. It can be activated in-hand to extend or retract razor blades which cause significant damage when thrown. Also heavily armored."
```
Comment (ReddicusDragon) [inline code/modules/clothing/head/jobs.dm:106]: won't this make it so that normal, non traitor players can read this description too?

Comment (Skrem7) [reply code/modules/clothing/head/jobs.dm:106]: The hat isn't metashielded and it's not at all obvious to people who don't know what it does

Comment (Moggram) [reply code/modules/clothing/head/jobs.dm:106]: It may need it, Sec let me keep it one time when I was in time out.

Comment (Skrem7) [reply code/modules/clothing/head/jobs.dm:106]: would require a lot more work and a separate pr which would have a different goal

**Review by MajManatee — APPROVED** [review]
hat

**Pull request merged**